json_parsers/json_parser4.py.py

```diff
@@ -0,0 +1,34 @@
+
+import json
+
+    
+class Json:
```

Comment:
Class name ideally should not be a built-in module names. And it should be followed camel case
Example
JsonReader